processor/compile/figure.js

```diff
@@ -1,9 +1,10 @@
 const { imgSizeByWidth } = require('../parse/magic-block-parser');
 
 const compileImage = image => {
-  const { className, width } = image.data.hProperties || {};
+  const { align, className, width } = image.data.hProperties || {};
   const img = {
     image: [image.url, image.title, image.alt],
+    ...(align && { alignment: align }),
```

Comment:
Can we keep the naming the same throughout? It's so much easier the closer `mdast` is to `slate` stuff.

Reply:
Totally, I think I was following the pattern with width where we have `width` and then `sizing`. But agree it's mighty confusing! Let me see if I can update both of them